management/migrations/0019_auto_20170716_1644.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2017-07-16 23:44
+from __future__ import unicode_literals
+
+import datetime
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('management', '0018_auto_20170714_1233'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='contact',
+            name='contact_id',
+            field=models.CharField(default="0", max_length=20),
+        ),
+        migrations.RemoveField(
+            model_name='contact',
+            name='contact_id',
+        ),
```

Comment:
^ This looks a bit messed up (adding and removing the same field) and I think the change should be in https://github.com/charityscience/csh-sms/pull/18/files.

Also I think there should be an id field in the schema for all the relevant data objects.